.github/workflows/ci.yml

```
@@ -33,4 +33,7 @@ jobs:
           SESSION_SECRET: ${{ secrets.SESSION_SECRET }}
           UPSTASH_REDIS_REST_URL: ${{ secrets.UPSTASH_REDIS_REST_URL }}
           UPSTASH_REDIS_REST_TOKEN: ${{ secrets.UPSTASH_REDIS_REST_TOKEN }}
+          FACEBOOK_APP_ID: ${{ secrets.FACEBOOK_APP_ID }} 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)